charts/qat/Chart.yaml

```
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: intel-device-plugins-qat
+description: A Helm chart for QAT Intel Device Plugin
```

nit: we should probably keep the descriptions consistent. SGX does not yet follow this pattern either but I'd suggest:
```suggestion
description: A Helm chart for Intel QAT Device Plugin
```